03-rnaseq/clustering_rnaseq_01_heatmap.Rmd

```diff
@@ -283,14 +284,15 @@ We are going to use the `rlog()` function from the `DESeq2` package to normalize
 For more information about these transformation methods, [see here](https://alexslemonade.github.io/refinebio-examples/03-rnaseq/00-intro-to-rnaseq.html#deseq2-transformation-methods). 
 
 ```{r}
-# Normalize the data in the `DESeqDataSet` object using the `rlog()` function from the `DESEq2` R package
+# Normalize the data in the `DESeqDataSet` object
+# using the `rlog()` function from the `DESEq2` R package
 dds_norm <- rlog(dds)
 ```
 
 ## Choose genes of interest
 
-Although you may want to create a heatmap including all of the genes in the set, alternatively, the heatmap could be created using only genes of interest.
-For this example, we will sort genes by variance, but there are many alternative criterion by which you may want to sort your genes <i>e.g.</i> fold change, t-statistic, membership to a particular gene ontology, so on. 
+Although you may want to create a heatmap including all of the genes in the dataset, this can produce a very large image that is hard to interpret.Alternatively, the heatmap could be created using only genes of interest.
```

Comment:
```suggestion
Although you may want to create a heatmap including all of the genes in the dataset, this can produce a very large image that is hard to interpret. 
Alternatively, the heatmap could be created using only genes of interest.
```